bench-run/benchs/cbench/run.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
```

Comment:
I would prefer to rename `bench-run` into `bench`, `benchs` into `run`.

Reply:
Ignored :)
Why?

Reply:
Ok, got it.